Tests/TokamakStaticHTMLTests/RenderingTests.swift

```diff
@@ -203,6 +203,31 @@ final class RenderingTests: XCTestCase {
     )
     #endif
   }
+
+  func testMaterial() {
+    #if compiler(>=5.5) || os(WASI)
```

Comment:
Just curious, what features of 5.5 are used here? Since we don't have the latest Xcode on GitHub Actions, these tests won't run there, right?

Reply:
It just needs the newest SwiftUI in Xcode 13 so it can find `Material`, and some new modifiers. I wasn't sure what the correct way to restrict it to that is besides using Xcode 13 on CI or restricting to the compiler version shipping with that Xcode version.

Reply:
Right, but these tests should be using `Material` from Tokamak, not SwiftUI. There's no `import SwiftUI` in `RenderingTests.swift` at all, and there's an explicit import of `TokamakStaticHTML`, so maybe removing this `#if compiler(>=5.5)` should be an issue? And `os(WASI)` shouldn't have any effect since snapshot tests aren't supported on that platform anyway, and there's a top-level platform check for macOS in that file.

Reply:
Oops, sorry. Mixed that up with the demo files which wouldn't build. I'll get rid of it.